pkg/tarmak/cluster/cluster.go

```diff
@@ -393,6 +419,11 @@ func (c *Cluster) Variables() map[string]interface{} {
 			output[fmt.Sprintf("%s_ami", instancePool.TFName())] = ids
 		}
 		output[fmt.Sprintf("%s_instance_count", instancePool.TFName())] = instancePool.Config().MinCount
+		if instancePool.Config().AllowCIDRs != nil {
```

Comment:
Not sure if I want to keep it open like this to all instance pools. At this moment this will also generate a tfvar for etcd, vault, ... in terraform.tfvars but we don't use them in terraform or in one of the terraform modules. 

Input wanted for this